src/app/models/pending_submission.server.model.ts

```diff
@@ -0,0 +1,40 @@
+import { Mongoose } from 'mongoose'
+import { SubmissionType } from 'shared/types'
+
+import { ISubmissionModel, ISubmissionSchema } from 'src/types'
+
+import {
+  EmailSubmissionSchema,
+  EncryptSubmissionSchema,
+  SubmissionSchema,
+} from './submission.server.model'
+
+export const PENDING_SUBMISSION_SCHEMA_ID = 'PendingSubmission'
+
+const compilePendingSubmissionModel = (db: Mongoose): ISubmissionModel => {
+  const PendingSubmission = db.model<ISubmissionSchema, ISubmissionModel>(
+    PENDING_SUBMISSION_SCHEMA_ID,
+    SubmissionSchema,
+  )
+  PendingSubmission.discriminator(SubmissionType.Email, EmailSubmissionSchema)
+  PendingSubmission.discriminator(
+    SubmissionType.Encrypt,
+    EncryptSubmissionSchema,
+  )
+  return db.model<ISubmissionSchema, ISubmissionModel>(
+    PENDING_SUBMISSION_SCHEMA_ID,
+    SubmissionSchema,
+  )
```

Comment:
I know this copies the code from `compileSubmissionModel()`, but why are we returning this call instead of `PendingSubmission`?

Reply:
Yeah, I think you might be right too. Let me update it and we can give it a shot!